src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -117,6 +120,28 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
             }
         }
 
+        //
+        public List<int> GetPortsToIgnoreFromAnnotations(V1ServiceList serviceNameSpace) {
+            var portsToIgnore = new List<int>();
+            foreach(var item in serviceNameSpace.Items) {
+                if (item.Metadata.Annotations != null) {

```

Comment:
We should probably add check that Metadata is not null as well:
if (item.Metadata != null && item.Metadata.Annotations != null)